types/webpack-dev-server/package.json

```
@@ -1,6 +1,7 @@
 {
   "private": true,
   "dependencies": {
+    "chokidar": "^3.5.2",
```

```suggestion
    "chokidar": "^3.5.1",
```
just to match the source